Lib/pathlib.py

```diff
@@ -1329,6 +1329,27 @@ def is_file(self):
             # (see https://bitbucket.org/pitrou/pathlib/issue/12/)
             return False
 
+    def is_mount(self):
+        """
```

Comment:
Easier, possibly. But I still think I'll hit my current issue with Windows. No like function in the library hits the 'os' module and I am pretty sure Python core developers would want this library standalone where possible.